asteroid/dsp/overlap_add.py

```diff
@@ -10,9 +10,15 @@ class LambdaOverlapAdd(torch.nn.Module):
     Segment input signal, apply lambda function (a neural network for example)
     and combine with OLA.
 
+    `LambdaOverlapAdd` works with `foo.separate` and the `asteroid-infer` CLI if
+    the given `nnet` implements `foo.Separatable`. (Both `asteroid.models.BaseModel`
+    and `asteroid.models.BaseEncoderMaskerDecoder` do.)
+
     Args:
         nnet (callable): Function to apply to each segment.
-        n_src (int): Number of sources in the output of nnet.
+        n_src (Optional[int]): Number of sources in the output of nnet.
+            If None, the number of sources is determined by the network's output,
+            but some correctness checks cannot be performed.
```

Comment:
This was the easiest (but maybe not the cleanest) way to make OLA work with any/unknown models, like in asteroid-infer. The alternative would be to add `n_src` to the `Separatable` interface (ie. the model)

Reply:
No because I didn’t want to break the interface

Reply:
Ok.